docs/command_reference.md

```diff
@@ -0,0 +1,279 @@
+<!--[metadata]>
++++
+title = "Notary Command Reference"
+description = "Notary command reference"
+keywords = ["docker, notary, command, notary command, reference"]
+[menu.main]
+parent="mn_notary"
+weight=99
++++
+<![end-metadata]-->
+
+# Notary Command Reference
+
+## Terminology Reference
+1. **GUN**: Notary uses Globally Unique Names (GUNs) to identify trusted collections.
+2. **Target**: Notary refers to the files the framework is used to distribute as "target files".  Target files are opaque to the framework. Whether target files are packages containing multiple files, single text files, or executable binaries is irrelevant to Notary.
+3. **Trusted Collection**: A trusted collection is a set of target files of interest to the application.
+4. **Notary Repository**: A notary repository refers to the set of signed metadata files that describe a trusted collection.
+5. **Key roles**:
+    - **Root**: The root role delegates trust to specific keys trusted for all other top-level roles used in the system.
+    - **Targets**: The targets role's signature indicates which target files are trusted by clients.
+    - **Delegations**: the targets role can delegate full or partial trust to other roles.  Delegating trust means that the targets role indicates another role (that is, another set of keys and the threshold required for trust) is trusted to sign target file metadata.
+    - **Snapshot**:  The snapshot role signs a metadata file that provides information about the latest version of all of the other metadata on the trusted collection (excluding the timestamp file, discussed below).
+    - **Timestamp**:  To prevent an adversary from replaying an out-of-date signed metadata file whose signature has not yet expired, an automated process periodically signs a timestamped statement containing the hash of the snapshot file.
+
+To read further about the framework Notary implements, check out [The Update Framework](https://github.com/theupdateframework/tuf/blob/develop/docs/tuf-spec.txt)
+
+## Command Reference
+
+### Set up Notary CLI
+
+Once you install the Notary CLI client, you can use it to manage your signing
+keys, authorize other team members to sign content, and rotate the keys if
+a private key has been compromised.
+
+When using the Notary CLI client you need to specify where is Notary server
+you want to communicate with with the `-s` flag, and where to store the private keys and cache for
+the CLI client with the `-d` flag.  There are also fields in the [client configuration](/reference/client-config.md) to set these variables.
+
+```bash
+# Create an alias to always have the notary client talking to the right notary server
+$ alias notary="notary -s <notary_server_url> -d <notary_cache_directory>
+```
+
+When working Docker Content Trust, it is important to connect to Docker Hub's Notary server located at `https://notary.docker.io`, and specify notary's metadata cache as `~/.docker/trust`.
```

Comment:
The `https:/notary.docker.io` here is not very accurate, if one enabled the DCT and set up his own Notary service, what he need to specify is the URL of his notary server(and it might be the most instance).